marconi-core/src/Marconi/Core/Experiment/Indexer/FileIndexer.hs

```diff
@@ -0,0 +1,296 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-deferred-out-of-scope-variables #-}
```

Comment:
Bin?

Reply:
Oh god yes! I was wondering why HLS had a weird behaviour and it was probably the reason... I'd probably fatfingered it using the HLS suggestions.